### DISCUSSION TITLE
AzureOpenAI metrics not sent

### DISCUSSION THREAD

**Discussion by nitin302** [traceloop/openllmetry#2107] [General]
We use AzureChatOpenAI() via from langchain_openai import AzureChatOpenAI

**Comment by nitin302** [comment]
Is this supported?

**Comment by nirga** [comment]
Hi @nitin302 sorry for the delay here! Which platform are you sending the data to?

**Comment by nitin302** [comment]
This relates to https://github.com/traceloop/openllmetry/issues/1912 - so I understand it is already on your roadmap, thanks
